server/libs/infra/src/entities/asset.entity.ts

```diff
@@ -102,6 +102,7 @@ export class AssetEntity {
   sharedLinks!: SharedLinkEntity[];
 
   @ManyToMany(() => AlbumEntity, (album) => album.assets)
+  @JoinTable()
```

Comment:
I don't think this is needed on both sides, just one, and it's on the album side already.